src/miner.cpp

```diff
@@ -182,14 +171,21 @@ class AddOutputsToCoinbaseTxAndSign
         return miner_reward + nFees;
     }
 
-    void ComputeBindingSig(rust::Box<sapling::Prover> saplingCtx, std::optional<orchard::UnauthorizedBundle> orchardBundle) const {
+    void ComputeBindingSig(rust::Box<sapling::Builder> saplingBuilder, std::optional<orchard::UnauthorizedBundle> orchardBundle) const {
```

Comment:
nonblocking: I'm a little surprised that the builder is passed in here instead of `saplingBundle` being computed by the caller and passed in; that seems like it would be more consistent with how `orchard` is handled.

Reply:
It's because I haven't finished the builder refactor, and Saplings proofs are still created by the builder rather than from the unauthorized bundle.